.github/workflows/validator-rs.yml

```diff
@@ -73,7 +73,6 @@ jobs:
           profile: minimal
           toolchain: stable
           override: true
-          components: rustdoc
       - uses: Swatinem/rust-cache@v1
```

Comment:
Why don't you use the official cache action: https://github.com/actions/cache/blob/main/examples.md#rust---cargo?

Reply:
Because (in no particular order):

 - it's a lot shorter/less cluttered/less magic-stringy than calling the cache action directly;
 - it doesn't require me to think about whether those cache keys and magic paths are correct because the defaults of this action should already be;
 - it's actually documented in the context of Cargo/Rust beyond just throwing an example out there and calling it a day;
 - it was the top hit on google;
 - I'm pretty sure I've seen it used somewhere before, though I'm not sure where.

Is there anything wrong with it?

Reply:
No, not necessarily anything wrong with it. But I'd like to suggest using offical actions (when available/possible) because it is more likely that they are being maintained.

Reply:
I mean... this action is just a thin wrapper around the official method, I'm pretty sure. So even if it's never updated, it's not worse than copypasting the (current) official method into the workflow directly. On the other hand, if ever something changes with cargo that requires changes to which directories need to be cached, there is at least a chance that the this third party action will be updated.